### PR TITLE
Lotame

### PR DESCRIPTION
**Pull request by prabrisha-rudder**
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/42)

### CODE REVIEW

**Comment by sayan-rudder** [comment]
let's also mention the brotli/gzip file size in each PR from now on.
https://github.com/ai/size-limit

File: integrations/Lotame/browser.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  }
+
+  compileUrl(map, url){
+
+    Object.keys(map).forEach(key => {
+      let replaceKey = "{{"+key+"}}";
+      let regex = new RegExp(replaceKey, 'gi');
+      url = url.replace(regex, map[key]);
+    });
+    console.log(url);
```
Comment (sayan-rudder) [inline integrations/Lotame/browser.js]: logger.debug?

File: integrations/Lotame/browser.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    if(window.LOTAME_SYNCH_CALLBACK && typeof window.LOTAME_SYNCH_CALLBACK == "function"){
+      window.LOTAME_SYNCH_CALLBACK();
+    }
+
+  }
+
+  compileUrl(map, url){
+
+    Object.keys(map).forEach(key => {
+      let replaceKey = "{{"+key+"}}";
```
Comment (sayan-rudder) [inline integrations/Lotame/browser.js]: check hasOwnPrperty

File: integrations/Lotame/browser.js (hunk 1 above)
Comment (prabrisha-rudder) [reply integrations/Lotame/browser.js]: forgot to remove it. not needed. removing it now.

File: integrations/Lotame/browser.js (hunk 2 above)
Comment (prabrisha-rudder) [reply integrations/Lotame/browser.js]: incorporated.

**Comment by sayan-rudder** [comment]
👍

**Comment by prabrisha-rudder** [comment]
> let's also mention the brotli/gzip file size in each PR from now on.
> https://github.com/ai/size-limit

brotli/gzip file size : 26KB

**Pull request merged**